payments/models.py

```diff
@@ -416,14 +408,12 @@ def save(self, *args, **kwargs):
             tax_percentage = self.product.tax_percentage
 
             if self.lease:
-                # If the lease doesn't last for the whole season
-                if not self.is_full_season():
-                    price = calculate_product_partial_season_price(
-                        price,
-                        self.lease.start_date,
-                        self.lease.end_date,
-                        summer_season=isinstance(self.lease, BerthLease),
-                    )
+                price = calculate_product_partial_season_price(
```

Comment:
what if we rename this function to `calculate_product_season_price` since it is not necessarily `partial` anymore?

Reply:
🤔 I have my doubts because it would still calculate the "partial" price as well as the "full" season price, and also to keep consistency with the other helper functions (partial month, and partial year)

Reply:
fair enough, consistency argument makes sense 👍